resources/io/jenkins/infra/docker/Makefile

```diff
@@ -34,9 +34,9 @@ lint: ## Lint the $(IMAGE_DOCKERFILE) content
 
 build: ## Build the Docker Image $(IMAGE_NAME) from $(IMAGE_DOCKERFILE)
 	@echo "== Building $(IMAGE_NAME) from $(call FixPath,$(IMAGE_DOCKERFILE)) with docker:"
-	docker buildx create --use
-    docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
-	docker buildx build \
+		docker buildx create --use
```

Comment:
Could you give a name to this "builder instance" and only create it if not existing?

Also, can you remove it after the `docker run` is finished with success (no need to take care of failures: that's what the condition would be there for)

Reply:
added in https://github.com/jenkins-infra/pipeline-library/pull/724/commits/75be361a5695998ba0f4f9e66d2da7c949d1c483

Reply:
removed as per rubber duck